WORKSPACE

```diff
@@ -51,58 +79,69 @@ _piptool_install()
 
 git_repository(
     name = "subpar",
-    remote = "https://github.com/google/subpar",
     # HEAD as of 2018/02/15
     commit = "1f695ee5d42585a66d9dd9b71219eb8551e59c89",
+    remote = "https://github.com/google/subpar",
 )
 
-# Test data for WHL tool testing.
+###################################
+# Test data for WHL tool testing. #
+###################################
+
 http_file(
     name = "grpc_whl",
+    downloaded_file_path = "grpcio-1.6.0-cp27-cp27m-manylinux1_i686.whl",
     sha256 = "c232d6d168cb582e5eba8e1c0da8d64b54b041dd5ea194895a2fe76050916561",
     # From https://pypi.python.org/pypi/grpcio/1.6.0
-    url = ("https://pypi.python.org/packages/c6/28/" +
-           "67651b4eabe616b27472c5518f9b2aa3f63beab8f62100b26f05ac428639/" +
-           "grpcio-1.6.0-cp27-cp27m-manylinux1_i686.whl"),
+    urls = [("https://pypi.python.org/packages/c6/28/" +
```

Comment:
Why is this constructed using string concatenation? I'd prefer just one long string, because that would make it clearer that there's only a single URL in the urls attribute.

Reply:
The multiline form has the advantage of being able to easily read the distinct pieces of the string, for instance that the wheel part of the URL matches the file path attr above. I'm not sure the value of having an unbroken URL is worth it in this case.

Reply:
SGTM.